### ISSUE TITLE
Questie 5.2.0: LUA Error when searching

### ISSUE THREAD

**Issue by Mankriks-Neighbor** [AeroScripts/QuestieDev#1526]
Every time i want to search with "Advanced Search" Questie throws an error:

```
Message: ...ace\AddOns\Questie\Modules\Journey\QuestieSearch.lua:142: bad argument #1 to 'lower' (string expected, got nil)
Time: Mon Nov 11 19:15:29 2019
Count: 1
Stack: ...ace\AddOns\Questie\Modules\Journey\QuestieSearch.lua:142: bad argument #1 to 'lower' (string expected, got nil)
[C]: in function `lower'
...ace\AddOns\Questie\Modules\Journey\QuestieSearch.lua:142: in function `Search'
...ace\AddOns\Questie\Modules\Journey\QuestieSearch.lua:35: in function `ByName'
...Ons\Questie\Modules\Journey\QuestieSearchResults.lua:393: in function `DrawSearchResultTab'
...Ons\Questie\Modules\Journey\QuestieSearchResults.lua:504: in function <...Ons\Questie\Modules\Journey\QuestieSearchResults.lua:501>
(tail call): ?
[C]: ?
[string "safecall Dispatcher[3]"]:9: in function <[string "safecall Dispatcher[3]"]:5>
(tail call): ?
...erface\AddOns\Questie\Libs\AceGUI-3.0\AceGUI-3.0.lua:314: in function `Fire'
...tie\Libs\AceGUI-3.0\widgets\AceGUIWidget-EditBox.lua:74: in function <...tie\Libs\AceGUI-3.0\widgets\AceGUIWidget-EditBox.lua:71>

Locals: <none>
```

I moved all addons except Questie out of the interface\addons Folder to see if it is another one of my addons but the error message still showed up.

**Comment by BreakBB** [comment]
Yeah we've already noticed this a fixed it in f952adfb6f69527da93eb1fb6d1a318a15dfc1f7 We will release a hotfix soon ™️ 

Moreover you don't have to move the addons our of the folder. It is enough if you simply disable them.

**Comment by drejjmit** [comment]
Fixed in 5.2.1
